src/components/Footer.astro

```diff
@@ -3,6 +3,11 @@ import { Image } from 'astro:assets'
 import textArrowIcon from '@assets/icons/text-arrow.svg?raw'
 import Default from '@astrojs/starlight/components/Footer.astro'
 import ThemeSelect from './ThemeSelect.astro'
+import { version } from '../../package.json'
+import { major, minor, patch } from 'semver'
+const [maj, min, pat] = [major(version), minor(version), patch(version)]
+const release = `${maj}.${min}.${pat}`
+const releaseLink = `https://github.com/daytonaio/daytona/releases/tag/v${release}`
```

Comment:
The version should always point to a minor release.

E.g. Docs can be on `0.24.8` while we only have `v0.24.1` released.

That's because the patch version of the docs is for content updates unrelated to the release while the patch version of Daytona is only for minor fixes.
I suggest the release link be `/v${release}.0` where `release = `maj.min`